ecs-agent/daemonimages/csidriver/driver/node.go

```diff
@@ -262,7 +291,8 @@ func newNodeService() nodeService {
 }
 
 func (d *nodeService) NodeUnstageVolume(ctx context.Context, req *csi.NodeUnstageVolumeRequest) (*csi.NodeUnstageVolumeResponse, error) {
-	klog.V(4).InfoS("NodeUnstageVolume: called", "args", *req)
+	klog.InfoS("NodeUnstageVolume: called", "args", *req)
+	t := time.Now()
```

Comment:
nit: could you make the variable name a bit more descriptive?

Reply:
Done